guides/examples/basic-web-app.md

```diff
@@ -148,13 +230,27 @@ ockam relay create db --to /node/db --at /project/default
 
 ### Connecting the web app
 
-With our database node now running, we need to connect a corresponding node on the web side. We'll start by creating another enrollment token, this time for a `component` labelled `web`:
+> In our story about our imaginary company and team of three, the following steps would
+> have been performed by Toby, our Ockam admin. Toby would then generate this one-time
+> enrollment ticket (just plain text) and then send that over to Zora. You can simply
+> follow these steps on your machine.
+
+With our database node now running, we need to connect a corresponding node on the web
+side. We'll start by creating another enrollment token, this time for a `component`
+labelled `web`:
 
 ```bash
 export WEB_TOKEN=$(ockam project ticket --attribute component=web)
 ```
 
-Next we'll create and enroll our node, set a policy to say it is only allowed to create inlet connections to the `db` component, and then finally we create that inlet:
+> In our story about our imaginary company and team of three, the following steps would
+> have been performed by Zora, our web app developer. Zora would use the one-time
+> enrollment ticket data (plain text) to create a node called 'web' and create a
+> [tcp-inlet](https://command.ockam.io/manual/ockam-tcp-inlet.html). You can simply follow
+> these steps on your machine.
+
+Next we'll create and enroll our node, set a policy to say it is only allowed to create
+inlet connections to the `db` component, and then finally we create that inlet:
 
 ```bash
 ockam identity create web
```

Comment:
I realize something in the relation between policies and inlets:

 - we create a policy associated to the `tcp-inlet` resource (on a given node)
 - then we create a specific `tcp-inlet` for a specific URL\

But we could create several `tcp-inlets` on the same node. Then how is it possible to set a different policy on each inlet?

Reply:
i am capturing this question on slack for further discussion